src/audits/RoleTooltipRequiresDescribedBy.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2012 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+goog.require('axs.AuditRules');
+goog.require('axs.browserUtils');
+goog.require('axs.constants.Severity');
+goog.require('axs.utils');
+
+/**
+ * 'Elements with role=tooltip should have at least one other element with aria-describedby referring to them.'
+ */
+axs.AuditRules.addRule({
+    name: 'roleTooltipRequiresDescribedby',
+    heading: 'Elements with role=tooltip should have a corresponding element with aria-describedby',
+    url: 'https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_02',
+    severity: axs.constants.Severity.SEVERE,
+    relevantElementMatcher: function(element) {
+        return axs.browserUtils.matchSelector(element, '[role=tooltip]');
```

Comment:
I would check for the element being visible at this point as well - as your (very nice) wiki text points out, the advice concerns a tooltip which is *displayed*.